packages/rsx/src/template_body.rs

```diff
@@ -81,8 +79,14 @@ pub struct TemplateBody {
     pub template_idx: DynIdx,
     pub node_paths: Vec<NodePath>,
     pub attr_paths: Vec<(AttributePath, usize)>,
+    pub dynamic_text_segments: Vec<FormattedSegment>,
     pub diagnostics: Diagnostics,
-    current_path: Vec<u8>,
+}
+
+impl Default for TemplateBody {
+    fn default() -> Self {
```

Comment:
maybe no default here - and just use this in the to_tokens directly? This might be confusing if we try to ::default() somewhere